Releasenotes.md

```diff
@@ -1,6 +1,76 @@
 # Release notes BRK-Bevragen
 
-## Versie 1.1.1:
+## **Versie 1.2.0:**
+
+### openapi.yaml
+
+- Algemeen
+  - Diverse verbeteringen in de documentatie (descriptions) en examples.
+  - Er is gebruikgemaakt van Haal-Centraal-common versie 1.2.0 (was 1.0.0)
+  - De default-waarde voor de Accept-Crs header is verwijderd.
+  - Uit alle componentnamen zijn de underscores verwijderd.
+
+
+- KadastraalOnroerendeZaak
+  - De default-waarde voor de Accept-Crs header is verwijderd bij /kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}
+  - PrivaatrechtelijkeBeperkingen, hypotheken en beslagen zijn toegevoegd aan de _embedded en _links. De bijbehorende identificaties zijn opgenomen in de KadastraalOnroerendeZaak.
```

Comment:
dit is niet waar. Alleen privaatrechtelijkeBeperkingen is toegevoegd aan _embedded

Reply:
Aangepast